test/e2e_new/kubetest/setup.go

```diff
@@ -0,0 +1,97 @@
+// +build e2e
+
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubetest
+
+import (
+	"flag"
+	"go/build"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/cluster-api/test/framework/clusterctl"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+var (
+	kubernetesRepoRoot string
+	configFilePath     string
+	useCIArtifacts     bool
+	flakeAttempts      int
```

Comment:
Need to delete this, not needed now, but also want to wait for the e2e to pass first.